mantidproject/mantid
# SumOverlappingTubes Algorithm

This is a new algorithm to combine detectors scans for the D2B powder diffractometer. Please read the algorithm documentation for a full description of behaviour.

**To test:**

1. Read docs, do they explain what the algorithm does?
1. Test with attached files. Try different options, and different cases such as choosing two files or a group workspace. Use a colour fill plot to see the output in the 2D cases.
1. Code review.

Fixes #20285.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
